utils/offers/offerAggregateOptions.js

```diff
@@ -7,26 +7,35 @@ const offerAggregateOptions = (query, params) => {
     proficiencyLevel,
     rating,
     language,
-    name,
+    search,
     languages,
     excludedOfferId,
     sort,
+    status = [],
```

Comment:
we don't need status to be an array, its a string

Reply:
fixed